### ISSUE TITLE
"Export to Quest" button doesn't work on (Debian) Linux

### ISSUE THREAD

**Issue by TheZipCreator** [Caeden117/ChroMapper#446]
**ChroMapper version**
0.8.523

**Describe the Issue**
Upon clicking "Export to Quest", an error is thrown and the progress is stuck at 0%

**Provide Screenshots or Videos**
https://paste.ee/p/zWMA9 generated bug report

**Steps to Reproduce**
* Install ADB via chromapper
* go to `/path/to/chromapper/quest-utils/platform-tools`
* chmod +x all executables in that directory (this should be done by default, and not doing this causes a different error)
* open a map in chromapper and press "Export to Quest"

**Expected Behavior**
the button should actually work

**Additional Information**
* This button worked on windows before I switched to linux

**Comment by Fernthedev** [comment]
Taking a glance at this, it seems to suggest you’re not using the Linux binary of CM? Why is it throwing Win32 exceptions on Linux? Wine?

That or it’s just the way Mono works on Linux?

also about the chmod +x error, that shouldn’t happen unless extracting the zip loses its permission attributes 🤔

**Comment by TheZipCreator** [comment]
yeah, I was confused about the "Win32Exception" too, since I am using the linux binary. I guess that's just the way Mono works.
also not sure why the permissions aren't saved, does the zip file that chromapper downloads have them? does the library you're using look at file permissions?

**Comment by MoistSac** [comment]
I don't have a quest to test with, but it seems that the adb command to get the device id worked, but the command to create a folder failed. The error log doesn't tell us what error the command made, so can you, from a command line, run:
```
/home/z/things/chromapper/chromapper/quest-utils/platform-tools/adb -s 1WMHH810UM0451 shell mkdir "\"sdcard/ModData/com.beatgames.beatsaber/Mods/SongLoader/CustomWIPLevels/The President is Dead\" -p -m 770
```
and see what the actual error is?

**Comment by TheZipCreator** [comment]
That seems to work fine. Something of note is that in the shell command, the quote is unbalanced. I'm not sure if this is actually a problem or not, but I had to fix it when typing in the command in bash

**Comment by ChrisJAllan** [comment]
Ah yup, [this function](https://github.com/Caeden117/ChroMapper/blob/fbf5a83789333346917cc99b2d0d90459eff845e/Assets/__Scripts/Singletons/Adb.cs#L205) generates unbalanced quotes. That should break export on all platforms, not just linux.  I'll let someone who can actually test if fix it.

**Comment by Fernthedev** [comment]
Double checking it, I'm not sure where the unbalanced code is. I might be blind but it seems to be fine. I can fix it in my #453, and if I did indeed make the mistake of having unbalanced quotes, sincere apologies.

Thanks
